docs/installation.rst

```diff
@@ -0,0 +1,47 @@
+Installation
+============
+
+.. code-block:: bash
+
+    $ pip install Flask-Reuploaded
+
+
+Migration guide from `Flask-Uploads`
+------------------------------------
+
+This package is a drop-in replacement of broken `flask-uploads`.
```

Comment:
Let's not call it broken... let's called it unmaintained... - and please capitalize `Flask-Uploads`